### ISSUE TITLE
`geojsonToArcGIS` has falsy checks for z-values

### ISSUE THREAD

**Issue by rgwozdz** [terraformer-js/terraformer#78]
[Checks for z-values are falsy](https://github.com/terraformer-js/terraformer/blob/main/packages/arcgis/src/geojson.js#L62-L64), and as result they don't evaluate as we'd hope if the z-value has a zero-value.

Imagine a coordinate array `[-170, 42, 0]` flowing through the following code:

```js
if (geojson.coordinates[2]) {
  result.z = geojson.coordinates[2];
}
```

Since the z-coordinate is `0` the check evaluates to false and the `result.z` is never set.

**Comment by rgwozdz** [comment]
Cool, one of us will pick it up.  Are you looking for additional maintainers?  Given that we are using it, we could help out.  Or if you want to move it to koop, that's always an option.

**Comment by jgravois** [comment]
> Are you looking for additional maintainers? Given that we are using it, we could help out.

of course. the more the merrier!

> move it to koop

I'm happy at the helm of the @terraformer-js ghost ship for now, but i'll definitely keep that offer in mind.

overall the code is pretty stable and external contributors like yourself are doing most of the work to pinpoint additional fixes and enhancements. my experience since moving it out of the @Esri org a couple years ago has been overwhelmingly positive.

**Comment by jgravois** [comment]
published as [`v2.1.1`](https://www.npmjs.com/package/@terraformer/arcgis/v/2.1.1). thank you both!
